faster_whisper/transcribe.py

```diff
@@ -483,15 +486,18 @@ def transcribe(
 
         # if no segment split is provided, use vad_model and generate segments
         if not vad_segments:
-            vad_segments = self.vad_model(
-                {"waveform": torch.from_numpy(audio).unsqueeze(0), "sample_rate": 16000}
-            )
-            vad_segments = merge_chunks(
-                vad_segments,
-                self.chunk_size,
-                onset=self.vad_onset,
-                offset=self.vad_offset,
-            )
+            if self.use_vad_model:
+                vad_segments = self.vad_model(
+                    {"waveform": torch.from_numpy(audio).unsqueeze(0), "sample_rate": 16000}
+                )
+                vad_segments = merge_chunks(
+                    vad_segments,
+                    self.chunk_size,
+                    onset=self.vad_onset,
+                    offset=self.vad_offset,
+                )
+            else:
+                raise RuntimeError("no vad_model found. Set 'use_vad_model' to True while loading the model")
```

Comment:
I think the first sentence of the error message should say "No vad segments found."